Time Tracker/Log.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+using nsButton;
+using nsCSV;
+
+namespace nsLog
+{
+    class Log
+    {
+        public Log(Stopwatch stopwatch)
+        {
+            this._stopwatch = stopwatch;
+        }
+
+        public void WriteTheLog(string elapsedTime, string name)
```

Comment:
Can I use the log for something else that is not only "elapsedTime"? If so, why do you call it like this?
Why dont you call this method just Write?